monitor.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
I don't think we want this file, do we? Is it included in the bundles, or is it just there as a helper (e.g. for make targets) ? afaict it should not be included in bundles

Reply:
No sorry, this was one of my test file.